src/ensembl/production/metadata/api.py

```diff
@@ -154,16 +132,16 @@ def __init__(self, metadata_uri=None, taxonomy_uri=None):
         super().__init__(metadata_uri)
 
         if taxonomy_uri is None:
-            taxonomy_uri = config.TAXONOMY_URI
+            taxonomy_uri = get_taxonomy_uri()
         self.taxonomy_db = load_database(taxonomy_uri)
```

Comment:
I believe you removed the `load_database` method, should it be `DBConnection` then?

Reply:
Yes, it should. I rolled back a good chunk of my taxonomy changes and I just wanted to push these changes to have a baseline to reintroduce my changes as all of the tests work at this point.

Reply:
Fully fixed